### ISSUE TITLE
go to definition on yaml flow mapping $ref

### ISSUE THREAD

**Issue by restjohn** [42Crunch/vscode-openapi#10]
I am not able to ctrl+click on a JSON Reference in a YAML [flow-style mapping](https://yaml.org/spec/1.2/spec.html#id2790832) node, e.g.,
```yaml
schema:
  type: array
  items: { $ref: '#/components/schemas/User' }
```
If I change to a normal indented mapping
```yaml
schema:
  type: array
  items: 
    $ref: '#/components/schemas/User'
```
then the ctrl+click go-to-definition works.

I am using VS Code version 1.36.1 on macOS.

Thanks for the nice extension.

**Comment by ak1394** [comment]
Thanks for reporting it! We expect to release a fix tomorrow.

**Comment by restjohn** [comment]
That would be great, thanks!

**Comment by ak1394** [comment]
Fixed in 1.8.8
